Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM ubuntu:18.04
+FROM ubuntu:latest
 
-LABEL maintainer="contact@securify.ch"
+LABEL maintainer="contact@chainsecurity.com"
```

Comment:
@ritzdorf We now redirect securify.ch to securify.chainsecurity.com, so I thought this made sense, let me know otherwise.